apps/condo/domains/billing/access/BillingReceiptFile.js

```diff
@@ -0,0 +1,77 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema billing.BillingReceiptFile 'file:File;context:Relationship:BillingIntegrationOrganizationContext:CASCADE;receipt:Relationship:BillingReceipt:CASCADE;controlSum:Text'`
+ */
+
+const get = require('lodash/get')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization,
+    queryOrganizationEmployeeFor,
+    queryOrganizationEmployeeFromRelatedOrganizationFor,
+} = require('../../organization/utils/accessSchema')
+const { STAFF } = require('../../user/constants/common')
+
+async function canReadBillingReceiptFiles ({ authentication: { item: user } }) {
```

Comment:
You need to add access for service users
example is here @condo/domains/billing/access/BillingProperty.js